src/utils/data/index.js

```diff
@@ -40,6 +42,8 @@ const FULL_BLOCK = '█';
 const LEFT_HALF_BLOCK = '▌';
 const RIGHT_HALF_BLOCK = '▐';
 const EMPTY_BLOCK = '-';
+const FIGURE_SPACE = ' ';
+const EN_SPACE = ' '
```

Comment:
Just out of curiosity, what is the difference between the EN and the Figure spaces?

Reply:
After some experiment, I've found FIGURE_SPACE does not work for Firefox and EN_SPACE is better. I don't know the difference in detail, but by definition EN_SPACE is 1/2 em. And FIGURE_SPACE should have the same size of digit zero of a fixed font. However, there are somebody saying that it's not guaranteed....